### PR TITLE
6723 - Searchfield: additional icon

### LINKED ISSUE #6723: Unit Testing: Create a Puppeteer Script for "Searchfield: Ability to add additional icon into searchfield and be able to run functions when clicked"
Describe the Issue
Create a unit testing for "Searchfield: Ability to add additional icon into searchfield and be able to run functions when clicked".

See attached BDD for more information.
[BDDStory_github#6453.docx](https://github.com/infor-design/enterprise/files/9300255/BDDStory_github.6453.docx)

Related github issue
#6453

### PR DESCRIPTION
**Pull request by jbrcna**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

**Related github/jira issue (required)**:
Closes #6723 
Related to #6453 

**Steps necessary to review your pull request (required)**:

1. pull this branch
2. build and run the app
3. run npm run e2e:puppeteer Searchfield
4. It should pass.

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [ ] A note to the change log.

### CODE REVIEW

File: test/components/searchfield/searchfield.puppeteer-spec.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const { checkInnerHTMLValue } = require('../../helpers/e2e-utils.js');
+
+describe('Searchfield', () => {
```
Comment (tmcconechy) [inline test/components/searchfield/searchfield.puppeteer-spec.js:3]:
```suggestion
describe('Searchfield Puppeteer Tests', () => {
```

File: test/components/searchfield/searchfield.puppeteer-spec.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  const baseUrl = 'http://localhost:4000/components/searchfield';
+
+  describe('custom icon button', () => {
```
Comment (tmcconechy) [inline test/components/searchfield/searchfield.puppeteer-spec.js:6]:
```suggestion
  describe('should support custom icon buttons', () => {
```

**Review by tmcconechy — COMMENTED** [review]
Likes good to me, just some naming suggestions

**Comment by jbrcna** [comment]
Thank you @tmcconechy

**Review by ericangeles — APPROVED** [review]

**Review by tmcconechy — APPROVED** [review]

**Pull request merged**